content/blog/posts/o3de-introducing-design-system.md

```diff
@@ -0,0 +1,95 @@
+
+The best way to make sure that tools in the editor are consistent would be to develop a complete set of design standards intended to manage at scale using reusable components and patterns. But first, let's talk about the problems our users was facing based off some research I done with people in the community.
+
+My first port of call was to talk to some the larger partner teams that had already contributed a bunch of great extensions to the engine, but as well as individual solo contributors that use O3DE as a side hobby. We asked questions like; "What things that you have contributed to O3DE?" and "Why was this feature a need for you (and or your team)?". 
+
+We discovered that a lot of our contributors were reinventing the wheel on problems that already had solutions. Not because they wanted to. It was because without any proper system in place to help guide the developers on how things should be done, this led to developers doing what felt best for them.
+
+This caused inconsistent replications of the same functionality for our workflows! Which isn't really scalable for an open source community. We knew that the intention for our contributors were positive, but it was just the lack of guidance missing to help put them on the right path. 
+
+So understandably some sort of unified framework for the cross-functional teams needed. This would help raise the quality bar of standards for consistency in O3DE. As the contributors lacked a central framework to align to, teams would just instinctively work on their own solo initiatives, further breeding inefficiencies. Something that is not ideal for a collaborative community.
+
+So to further ensure that the editor upheld these high quality of these standards, we developed some tenets that contributors should follow:
+
+![Diagram showing the tenets of the O3DE design system](/images/blog/intro-to-bjds/tenets-diagram.jpg)
+
+* **Efficiency:** User interactions and workflows should value efficiency and ease of use.
+* **Accessibility:** Accessibility options, such as color contrasts, fonts and icon legibility, interactions and scalability etc. are built into our the tool from the first step.
+* **Familiarity:** Align to our standards of the visual interfaces, tools, branding, and UI elements. No need to reinvent the wheel and confuse users.
+* **Modularity:** The key to each element of O3DE, including the interface design and component development.
+
+These tenets were formed to be straightforward and easy to remember. We want to ensure that the BJDS tenets are flexible for the community to embrace, and to employ through their contributions to the engine.
+
+### What's included in the BlueJay Design System
+
+**The BlueJay Design System** is made up of a set of standardized components and layouts to help accelerate the creation of new workflows for the editor. It also has [supporting documentation](https://o3de.org/docs/tools-ui/) on the O3DE Learn website, which details the following:
```

Comment:
- To be clear on terminology between blog posts and O3DE docs, use "O3DE Editor" for first appearance, then "Editor" for subsequent appearances. If you're referring to the many editors in O3DE, then you can say  "editors". 
```suggestion
**The BlueJay Design System** is made up of a set of standardized components and layouts to help accelerate the creation of new workflows in the Editor. It also has [supporting documentation](https://o3de.org/docs/tools-ui/) on the O3DE Learn website, which details the following:
```